src/openrct2/network/network.cpp

```diff
@@ -264,7 +264,7 @@ bool Network::BeginClient(const char* host, uint16 port)
 	return true;
 }
 
-bool Network::BeginServer(uint16 port, const char* address)
+bool Network::BeginServer(uint16 port)
```

Comment:
@IntelOrca Which caller? `network_begin_server` or even the calls to that? Because checking for the nullptr outside of the network code would require copying this check to multiple locations.